terraform/environments/cdpt-chaps/database.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_db_instance" "database" {
   password               = data.aws_secretsmanager_secret_version.db_password.secret_string
   vpc_security_group_ids = [aws_security_group.db.id]
   depends_on             = [aws_security_group.db]
-  snapshot_identifier    = "arn:aws:rds:eu-west-2:613903586696:snapshot:cdpt-dev-staging-snapshot-9-1-24"
```

Comment:
Do we need to set this per environment?

Reply:
I guess not, it gets saved in the snapshots for each env separately. It could just get called 'final-snapshot'